resources/clients/react/elros-pam/src/LoginNavigation.tsx

```diff
@@ -14,9 +15,7 @@ function LoginNavigation({ title }: LoginNavigationProps) {
   const { authStatus, currentUser, signOut } = useAuthStore();
   const { clearTags } = useTagsStore();
 
-  const signedInUtilities = (
-    currentUser: User | null
-  ): TopNavigationProps.Utility => ({
+  const signedInUtilities: TopNavigationProps.Utility = {
```

Comment:
I guess we don't have to make it a function because this will get re-executed when the user changes anyway?

Reply:
Yep!